web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -241,6 +248,85 @@ fixmystreet.update_list_item_buttons = function($list) {
   $list.children(':last-child').find('[name="shortlist-down"]').prop('disabled', true);
 };
 
+fixmystreet.pageController = {
+    toPage: function(page, opts) {
+        var $curr = $('.js-reporting-page--active');
```

Comment:
Why have this plus opts.current?

Reply:
A good question! Umm, I wonder if I had a reason, or if I just added things together and didn't spot. Let's go with the latter.